lib/whenever/integer.rb

```diff
@@ -0,0 +1,13 @@
+class Integer
```

Comment:
Should probably extend `Numeric` instead so these sugar methods are available on all types.